app/src/main/java/com/onboarding/wheatherapp/MainActivity.kt

```diff
@@ -17,7 +18,8 @@ class MainActivity : AppCompatActivity() {
         super.onCreate(savedInstanceState)
         binding = ActivityMainBinding.inflate(layoutInflater)
         setContentView(binding.root)
-        presenter = WeatherAppPresenter(WeatherAppModel(), WeatherAppView(this, binding))
+        presenter =
+            WeatherAppPresenter(WeatherAppModel(WeatherService()), WeatherAppView(this, binding))
```

Comment:
please update hard wrap to 140 chars at file > settings > editor > code style

Reply:
Solved!

Reply:
also this can fit on a single line

Reply:
Solved!